### PR TITLE
wd_demo.au3 + GUISetBkColor

### PR DESCRIPTION
**Pull request by mlipok**
On my Windows 11 I do no see differences between checked and unchecked Buttons, by adding GUISetBkColor() it becomes more contrasting / distinguishable.

### CODE REVIEW

File: wd_demo.au3 [hunk 1]
```diff
@@ -55,7 +56,7 @@ Func _WD_Demo()
 	Local $aCheckboxes[$iCount]
 
 	Local $hGUI = GUICreate("Webdriver Demo", 200, 150 + (20 * $iCount), 100, 200, BitXOR($GUI_SS_DEFAULT_GUI, $WS_MINIMIZEBOX))
-
+	GUISetBkColor($COLOR_LIGHTSTEELBLUE)
```
Comment (Danp2) [inline wd_demo.au3:59]: This doesn't compile for me. Are you running beta? I don't see this constant in ColorConstants.au3

**Review by Danp2 — CHANGES_REQUESTED** [review]

File: wd_demo.au3 (hunk 1 above)
Comment (Danp2) [reply wd_demo.au3:59]: Confirmed that $COLOR_LIGHTSTEELBLUE is available in latest beta, but we want this to work without requiring beta. Can you retest using a different color, such as $CLR_SILVER or $CLR_MEDGRAY?

**Comment by mlipok** [comment]
btw. I find an issue when I was trying different colors then I found that some for example: $CLR_MEDGRAY has been removed form Beta... (at least from dev version of AutoIt Include).

**Comment by mlipok** [comment]
current:
image

proposal:
image

**Pull request merged**